.travis.yml

```diff
@@ -41,7 +41,7 @@ script:
 - cd ..
 - npm run test-base
 - set +e # (we deactivate set -e here, to prevent sudden log truncation in Travis, as it was suggested by the support team)
-- node run-tests --js --python 5; export TESTS_FAILED=$?
+- node run-tests --js --python 5 2>&1; export TESTS_FAILED=$?
```

Comment:
Yep, as you see, redirecting stderr and stdio doesn't really help when Python is launched from JavaScript: https://travis-ci.org/ccxt/ccxt/builds/648720270#L2290